workflow-service/src/test/java/com/redhat/parodos/workflow/execution/service/WorkFlowServiceImplTest.java

```diff
@@ -0,0 +1,298 @@
+package com.redhat.parodos.workflow.execution.service;
+
+import com.redhat.parodos.workflow.WorkFlowDelegate;
+import com.redhat.parodos.workflow.WorkFlowStatus;
+import com.redhat.parodos.workflow.definition.entity.WorkFlowDefinition;
+import com.redhat.parodos.workflow.definition.repository.WorkFlowDefinitionRepository;
+import com.redhat.parodos.workflow.execution.entity.WorkFlowExecution;
+import com.redhat.parodos.workflow.execution.entity.WorkFlowTaskExecution;
+import com.redhat.parodos.workflow.execution.repository.WorkFlowRepository;
+import com.redhat.parodos.workflow.execution.repository.WorkFlowTaskRepository;
+import com.redhat.parodos.workflow.task.BaseWorkFlowTask;
+import com.redhat.parodos.workflow.task.WorkFlowTaskStatus;
+import com.redhat.parodos.workflows.work.*;
+import com.redhat.parodos.workflows.workflow.SequentialFlow;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class WorkFlowServiceImplTest {
+
+	private WorkFlowDefinitionRepository wfDefRepo;
```

Comment:
In similar way we have naming conventions [here](https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names)